fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -103,18 +103,36 @@ def self.print_table(actions)
     # Lane chooser if user didn't provide a lane
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
+      available = ff.runner.lanes[platform].to_a
+      if available.empty?
+        UI.error "It looks like you don't have any lanes to run just yet. Check out how to get started here: https://github.com/fastlane/fastlane 🚀"
+        exit
+      end
+
+      rows = []
+      available.each_with_index do |lane, index|
+        rows << [index + 1, lane.last.pretty_name, lane.last.description.first]
+      end
+
+      rows << [rows.size + 1, "cancel", "No selection!"]
```

Comment:
I wonder if cancel should be with value `0` instead, that way it's always the same :)